src/PID.c

```diff
@@ -0,0 +1,22 @@
+#include <main.h>
+
+/**
+ Default PID struct constructor, used for default parameter value
+ These specific values represent a smooth curve with no overshoot
+ */
+const struct PID DEFAULT_PID = { 5, 3, 3 };
+
+/**
+ Gets the output of the input passed through the PID controller for the delta time interval
+ */
+int pid(int input, int *errorSum, const unsigned long * const deltaTime, int target, PID pid) {
+    // Generate error signal
+	int error = target - input;
+	*errorSum += error;
```

Comment:
As in `*errorSum += error * deltaTime;`  or  just `*errorSum = error * deltaTime` ?